tests/modules/amplify/predict/main.nf

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PRODIGAL } from '../../../modules/prodigal/main.nf' addParams( options: [:] )
+include { AMPLIFY_PREDICT } from '../../../../modules/amplify/predict/main.nf' addParams( options: [:] )
+
+workflow amplify_predict {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
+    ]
+
+    PRODIGAL ( input, "gff" )
```

Comment:
im curious as to where the `gff` file is being sourced from?

Reply:
That's just specifying the output format for PRODIGAL (a require input for that module I guess)